my-app/src/test/java/com/JavaWebServer/app/ServerSocketTest.java

```diff
@@ -0,0 +1,51 @@
+package com.JavaWebServer.app;
+
+import java.net.Socket;
+import java.io.*;
+import static org.junit.Assert.*;
+import org.junit.*;
+
+public class ServerSocketTest {
+
+  @Test
+  public void TestServerWrapperisclosed() throws Exception {
+    ServerSocket testServer = new ServerSocket(9056 );
+    assertFalse(testServer.isClosed());
+     testServer.close();
+  }
+
+  @Test
+  public void TestServerclose() throws Exception {
+    ServerSocket testWrapper = new ServerSocket(9056);
+    testWrapper.close();
+    assertTrue(testWrapper.isClosed());
+  }
+
+
+  class MockServerSocket extends java.net.ServerSocket  {
```

Comment:
If you are going to have an internal class, you should either put it at the top or the bottom of your class depending on it's access level.